pkg/apis/scheduling/config/types.go

```diff
@@ -45,6 +45,34 @@ type QoSAwareNodeResourcesBalancedAllocationArgs struct {
 	ReclaimedResources []kubeschedulerconfig.ResourceSpec `json:"reclaimedResources,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourceTopologyArgs holds arguments used to configure the NodeResourceTopologyMatch plugin
+type NodeResourceTopologyArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// ScoringStrategy a scoring model that determine how the plugin will score the nodes.
+	ScoringStrategy *ScoringStrategy `json:"scoringStrategy,omitempty"`
+
+	// AlignedResources are resources should be aligned for dedicated pods.
+	AlignedResources []string `json:"alignedResources,omitempty"`
+
+	// ResourcePolicy are QRMPlugin resource policy to allocate topology resource for containers.
+	ResourcePolicy string `json:"resourcePolicy,omitempty"`
```

Comment:
maybe it's better to define a new type for ResourcePolicy instead of using `string` directly; and we should move the definition into `pkg/const` dir and make sure the QRM plugis in `katalyts-core` ref this too

Reply:
done.